src/zeal/docset.py

```diff
@@ -72,10 +74,13 @@ def remove(docset_name: str, docset_dir: str = config.docset_dir):
         raise exceptions.DocsetNotInstalledError(
             f"The docset to remove '{docset_name}' cannot be removed because it is not installed on your system."
         )
-    shutil.rmtree(os.path.join(docset_dir, f"{docset_name}.docset"))
-
+    if platform.system() == "Windows":
+        # windows performance of shutil.rmtree is pathetic
+        subprocess.run(["rmdir", "/q", "/s", str(Path(docset_dir, f"{docset_name}.docset").resolve())], check=True, shell=True)
+    else:
+        shutil.rmtree(str(Path(docset_dir, f"{docset_name}.docset").resolve()))
 
-def get_docset_versions(docset_name: str, feeds_dir: str):
+def get_docset_versions(docset_name: str, feeds_dir: Path):
     """Returns a list of available versions of a particular docset.
 
     :param docset_name: String, the name of the docset to remove
```

Comment:
Line 87: param `docset_name` is no longer a string, should be `pathlib.Path`

Reply:
docset_name is a str still